tools/public_api_guard/material/select.md

```diff
@@ -126,17 +126,29 @@ export class MatSelect implements AfterContentInit, OnChanges, OnDestroy, OnInit
     get multiple(): boolean;
     set multiple(value: boolean);
     // (undocumented)
-    static ngAcceptInputType_disabled: unknown;
+    static ngAcceptInputType_disabled:
+    /** Whether or not the overlay panel is open. */
```

Comment:
This change is weird, but I guess it's a problem with the API golden tool.

Reply:
Yeah, API extractor is very likely the problem.